discord/ext/commands/help.py

```diff
@@ -1005,9 +992,19 @@ class DefaultHelpCommand(HelpCommand):
         user if :attr:`dm_help` is set to ``None``. Defaults to 1000.
     indent: :class:`int`
         How much to indent the commands from a heading. Defaults to ``2``.
+    arguments_heading: :class:`str`
+        The arguments list's heading string used when the help command is invoked with a command name.
+        Useful for i18n. Defaults to ``"Arguments:"``
+
+        .. versionadded:: 2.0
     commands_heading: :class:`str`
         The command list's heading string used when the help command is invoked with a category name.
         Useful for i18n. Defaults to ``"Commands:"``
+    default_argument_description: :class:`str`
+        The default argument description string used when the argument's :attr:`~.commands.Parameter.description` is ``None``.
+        Useful for i18n. Defaults to ``"No description given."``
+
+        .. versionadded:: 2.0
```

Comment:
Surely if no description is given then it should just be empty?

Reply:
Correct but then the default (if any) looks out of place.
```
  name (default: ...)
  name - Description 
  name - Description (default: ...)
  name (default: ...)
```